packages/components/src/components/toast/toast/toast-utils.ts

```diff
@@ -1,2 +1,2 @@
-export const TOAST_STATES = ['neutral', 'success'] as const;
+export const TOAST_STATES = ['info', 'success'] as const;
```

Comment:
`neutral` must be deprecated and being mapped to `info` for backwards compatibility reasons.